### PR TITLE
Adding Hooks in docs

### PR DESCRIPTION
**Pull request by charliewilco**
Adds React Hooks as examples in docs #1987

### CODE REVIEW

**Comment by mrkev** [comment]
The second example works, but I tried the first one on CodeSandbox and it [didn't quite work](https://codesandbox.io/s/l5vx5v2ql7?fontsize=14). I'm happy to add it if we can get this going.

**Comment by charliewilco** [comment]
doh! `editor.current.focus()`

**Comment by mrkev** [comment]
ah, classic! 😄 Alright, I think this looks good to go.

File: README.md [hunk 1]
```diff
@@ -91,6 +91,38 @@ ReactDOM.render(
...
+
+  const editor = React.useRef(null)
+
+  function focusEditor() {
+    editor.current.focus();
+  }
+
+  React.useEffect(() => {
+    focusEditor()
+  }, [])
```
Comment (claudiopro) [inline README.md]: Nit: missing semicolon

File: README.md [hunk 2]
```diff
@@ -91,6 +91,38 @@ ReactDOM.render(
...
+Since the release of React 16.8, use can use [Hooks](https://reactjs.org/docs/hooks-intro.html) as a way to work with `EditorState` without using a class.
+
+
+```js
+function MyEditor() {
+  const [editorState, setEditorState] = React.useState(
+    EditorState.createEmpty()
+  );
+
+  const editor = React.useRef(null)
```
Comment (claudiopro) [inline README.md]: Nit: missing semicolon

**Comment by charliewilco** [comment]
should be resolved, sorry about that! i thought prettier would do js in markdown blocks https://prettier.io/blog/2017/11/07/1.8.0.html

**Pull request closed without merging**